src/app/components/FeaturedTranscripts.tsx

```diff
@@ -1,3 +1,5 @@
+"use server";
```

Comment:
Components are server components by default. Hence they do not require the "use server" directive.

However, implementing this change locally exposed some issues, after going down a rabbit hole of debugging, the problem stemmed from barrel exporting server and client components from a single entry point `src/app/components/index.tsx`.

Without marking as "use server" we get the long page download (18mb).

A fix would be segregating and grouping the imports, ideally these components should be under a sub-directory for incremental building. i.e a `components/layout` containing `header` and `footer` components and a couple other sub-directories.

Reply:
Actually, I added the "use server" directive because it prevented the long page download of (18mb) which was the major issue I wanted to fix, but I'll like to know the other issues this change exposed locally to be better informed.

To be clear, you're asking for a change to the export pattern of the components we're currently using to something like having different files components are exported from.

**Example** :
- components/server
- components/client
- components/layout

Reply:
Another issue noticed by this export is in `ExploreTranscripts` there's also a "use server" for `ExploreTranscripts` and removing that throws an error saying `fs module not found` which isn't right as it should be a server component being imported into `src/app/page.tsx` a server-rendered page.

Yes, a change in export pattern.
`components/server`, `component/client` won't be the right abstraction as some files are colocated and more reasonable if they are somewhat in the same directory.
E.g `ExploreTranscripts` and `ExploreTranscriptsClient`.

Example: 
- components/layout/header
- components/layout/footer
- components/layout/wrapper
- components/landing/...allTranscripts

This should come in after `add-hugo` PR

Reply:
Alright, I refactor the component imports not to use a single entry point and group colocated files accordingly after the `add-hugo` PR

Reply:
@IgboPharaoh kindly add the refactoring in this PR

cc: @0tuedon get ready for a rebase 😌. #15 is in staging so these refactors will be added to this PR and then merged to staging.

Reply:
poor me 🥲🥲